### PR TITLE
Revert to Swift 4.2

### LINKED ISSUE #144: carthage update --platform ios . error
Building scheme "RxAlamofire iOS" in _.xcodeproj
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -project /Users/chenxufeng/Documents/work/apub-companion-ios/Carthage/Checkouts/RxAlamofire/_.xcodeproj -scheme RxAlamofire\ iOS -configuration Release -derivedDataPath /Users/chenxufeng/Library/Caches/org.carthage.CarthageKit/DerivedData/10.1_10B61/RxAlamofire/4.4.1 -sdk iphoneos ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive -archivePath /var/folders/q3/6zscgh7134d_b15cxfkrqpfw0000gn/T/RxAlamofire SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO STRIP_INSTALLED_PRODUCT=NO (launched in /Users/chenxufeng/Documents/work/apub-companion-ios/Carthage/Checkouts/RxAlamofire)

### PR DESCRIPTION
**Pull request by rynecheow**
To resolve #144 until RxSwift 5.0.0 is released

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ All notable changes to this project will be documented in this file.
 
 #### Fixed
 
+## 4.5.0
+
+#### Fixed
+*  #144 until RxSwift 5.0.0 is released
```
Comment (sunshinejr) [inline CHANGELOG.md]: how about something like "Updated Carthage minimum Swift version to 4.2"?

Comment (freak4pc) [reply CHANGELOG.md]: I think it's just "Minimum Swift version is 4.2 (More details in #144)"

Comment (sunshinejr) [reply CHANGELOG.md]: Yeah, anything that would say what was updated without me needing to go to the GitHub issue to find out.

**Comment by rynecheow** [comment]
PR updated!

**Comment by rynecheow** [comment]
@freak4pc @sunshinejr Is this PR good to go yet?

**Comment by sunshinejr** [comment]
@rynecheow yeah I think so 👍🏻

**Pull request merged**